kubectl_deploy/postgres/service.yaml

```diff
@@ -0,0 +1,11 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: postgres-service
```

Comment:
I'd say the `service` part of the name here is a bit redundant, as the resource type is `Service`. It's also not a bad idea to indicate _what_ this postgres instance is for, so I'd go with `django-postgres` here.

Reply:
Addressed under 54a9bda.